src/main/java/com/synopsys/integration/alert/workflow/scheduled/PhoneHomeTask.java

```diff
@@ -100,6 +101,19 @@ public PhoneHomeTask(TaskScheduler taskScheduler, AboutReader aboutReader, Defau
         this.blackDuckProperties = blackDuckProperties;
     }
 
+    @Override
+    public boolean shouldScheduleTask() {
+        Map<String, String> environmentVariables = System.getenv();
+        Boolean skipPhoneHome = BooleanUtils.toBoolean(environmentVariables.get(PhoneHomeClient.SKIP_PHONE_HOME_VARIABLE));
```

Comment:
Is this alright? Or should I add an @Value to this class to get this variable?

Reply:
I'm in favor of the `@Value` even if it feels like overkill.